templates/taxi/manufacturer_form.html

```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+
+{% load crispy_forms_filters %}
+{% block content %}
+  <h1>{{ object|yesno:"Update,Create" }} Manufacturer</h1>
+  <form action="" method="post" novalidate>
+    {% csrf_token %}
+    {{ form|crispy }}
+    <input class="btn btn-primary" type="submit" value="Submit">
+  </form>
```

Comment:
Consider adding a 'Cancel' button to allow users to return to the previous page without submitting the form. This is in accordance with the checklist requirement: 'Add `Cancel` button apart from `Delete` one. The `Cancel` button will lead to the previous page the user was on.'